.env.example

```diff
@@ -3,8 +3,8 @@
 # OPENAI_API_KEY=
 OPENAI_CHAT_MODEL=gpt-4
 PORT=3000
-PGUSER=
-PGHOST=
-PGPASSWORD=
-PGDATABASE=
-PGPORT=
```

Comment:
There are the default values to use as `new Client()` or `new Pool()`: https://stackabuse.com/using-postgresql-with-nodejs-and-node-postgres/